### PR TITLE
Fb ww3 doc

### PR DESCRIPTION
**Pull request by aliabdolali**
The documentation for WW3 fix files, model driver and outputs are described in this PR.

### CODE REVIEW

**Comment by aliabdolali** [comment]
@DeniseWorthen I could not see the tables when I look at the preview. Could you take a look and let me know if you have a suggestion how to fix it?

File: doc/UsersGuide/source/InputsOutputs.rst [hunk 1]
```diff
@@ -325,21 +325,124 @@ WW3
...
+     - ✔
+   * - rmp_src_to_dst_conserv_002_001.nc
+     - Conservative remapping gsh_15m to gnh_10m
+     - na
+     - ✔
+   * - rmp_src_to_dst_conserv_003_001.nc
+     - Conservative remapping aoc_9km to gnh_10m
+     - na
+     - ✔
+     -
```
Comment (DeniseWorthen) [inline doc/UsersGuide/source/InputsOutputs.rst]: remove this line

File: doc/UsersGuide/source/InputsOutputs.rst [hunk 2]
```diff
@@ -325,21 +325,124 @@ WW3
...
+     - ✔
+   * - mod_def.glo_2deg
+     - Global grid [85S 85N]
+     - 2 degree
+     - ✔
+   * - mod_def.glo_gwes_30m
+     - Global NAWES 30 min wave grid [80S 80N]
+     - 30 min
+     - ✔
      -
```
Comment (DeniseWorthen) [inline doc/UsersGuide/source/InputsOutputs.rst]: remove this line

File: doc/UsersGuide/source/InputsOutputs.rst [hunk 3]
```diff
@@ -325,21 +325,124 @@ WW3
...
+     - Conservative remapping aoc_9km to gnh_10m
+     - na
+     - ✔
+     -
+
+
+.. _WW3_FixFilesp:
+
+.. list-table:: *Input grid information for single global configurations*
+   :widths: 35 35 30 15 15
```
Comment (DeniseWorthen) [inline doc/UsersGuide/source/InputsOutputs.rst]: You have 1 too many columns defined. You only want 4, not 5

File: doc/UsersGuide/source/InputsOutputs.rst [hunk 4]
```diff
@@ -325,21 +325,124 @@ WW3
...
+     - Number of grids defining input fields
+     - FUNIPNT
+     - Flag for using unified point output file.
+   * - IOSRV 
+     - Output server type 
+   * - FPNTPROC 
+     - Flag for dedicated process for unified point output
+   * - FGRDPROC
+     - Flag for grids sharing dedicated output processes
+     -
```
Comment (DeniseWorthen) [inline doc/UsersGuide/source/InputsOutputs.rst]: Remove this line

File: doc/UsersGuide/source/InputsOutputs.rst [hunk 5]
```diff
@@ -325,21 +325,124 @@ WW3
...
+The model driver input (ww3_multi.inp) includes the input, model and output grids definition, the starting and ending times for the entire model run and output types and intervals. The ww3_multi.inp.IN template is located under tests/parm/ directory. The inputs are described hereinafter:
+
+.. _WW3_Driver:
+
+.. list-table:: *Model driver input*
+   :widths: 30 70 
+   :header-rows: 1
+   * - NMGRIDS 
+     - Number of wave model grids
+   * - NFGRIDS
```
Comment (DeniseWorthen) [inline doc/UsersGuide/source/InputsOutputs.rst:422]: You have a conflicting number of columns. You're defining two colums, but you're listing 4 things before starting a new column (the * line). Should FUNIPNT be a new column?

File: doc/UsersGuide/source/InputsOutputs.rst (hunk 1 above)
Comment (aliabdolali) [reply doc/UsersGuide/source/InputsOutputs.rst]: Done

**Comment by aliabdolali** [comment]
@DeniseWorthen awesome, I can see the tables now. I think I am done.

**Comment by DeniseWorthen** [comment]
@aliabdolali I have a couple of questions about the wave information. 

First, there hasn't been anything added to section 4.1.4.2. That is the "grid description and IC files". It seems that information might be really contained in the previous section "static (fix) files" ? Should some of the information be moved to the grid/IC file section?

Secondly, Table 4.8 and 4.9 shows all items are date dependent. Is that correct? That column should indicate that, for example, the IC or the static file is for a model specific starting date.

**Comment by aliabdolali** [comment]
> @aliabdolali I have a couple of questions about the wave information.
> 
> First, there hasn't been anything added to section 4.1.4.2. That is the "grid description and IC files". It seems that information might be really contained in the previous section "static (fix) files" ? Should some of the information be moved to the grid/IC file section?
> 
> Secondly, Table 4.8 and 4.9 shows all items are date dependent. Is that correct? That column should indicate that, for example, the IC or the static file is for a model specific starting date.

The grid and model parameters are all defined in the fix files as they all are inside the mod_def files. Do we want to separate them? not all the models have the same structures, so I thought it is the best way to avoid confusion, but can change it and make it coherent with other model components. 
I do not know which tests use ICs, so we might need to ask @JessicaMeixner-NOAA to provide some info. Thanks for catching it. 
The items in the tables are date dependent in the sense that we referr  to a specific cdate in rt.sh for the WW3 model version. Other than that,. they are not date-dependent.

**Comment by JessicaMeixner-NOAA** [comment]
@aliabdolali -- You can determine which regression tests for WW3 use IC files by looking at the test descriptions of each of the regtests with waves or by running the regtests with waves, saving the run directories and looking for the restart files there.  I cannot be the only person who knows WW3 tests, you need to know this information too.   That being said, none of the WW3 regression tests have ICs.  I thought this documentation was more general than just these regression tests, so wouldn't just general information about WW3 ICs be relevant?

**Comment by aliabdolali** [comment]
it was my understanding as well but wanted to make sure. I have explained how WW3 dumps restart files. The descriptions are quite general, referring to the existing tests as examples. I think it is good for now, and there is room for improvement as we go.
@DeniseWorthen Could you remove the last column (date dependent) as it is a bit misleading. I have explained in the text that the WW3 mod_def files are version-dependent.

**Pull request merged**